frontend/src/types/Trip.ts

```diff
@@ -1,7 +1,7 @@
 import Location from "./Location";
 
 type Trip = {
-    id: string,
+    id?: string,
```

Comment:
Ich bekomme auf der Dashboard Page jetzt den Fehler: 
```
VM94 react_devtools_backend.js:4012 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `LocationMarkerGroup`. See https://reactjs.org/link/warning-keys for more information.
```
Ich glaube, weil in der LocationMarkerGroup die id der Location als key im List element benutzt wird und wir hier jetzt definiert haben, dass id optional is. 

Ich glaub wir haben jetzt zwei Möglichkeiten: 
1. auf der LocationMarkerGroup sowas zu machen `key={location.id ?? location.name}` - also die id nehmen und wenn die nicht da is den namen. 
2. für das POST separate Types `TripForm` und `LocationForm` ohne id attribute machen und für die normalen Types das id wieder mandatory setzen. 

Was meinst du?

Reply:
Bei 1.Option musst wahrscheinlich noch absichern, beim add location keine Location versehentlich 2 mal hinzugefügt wird
Bei 2.Option haben die Location noch kein id bevor Trip gespeichert wird, musst  key dann mit location.name gesetzt, d.h wieder gleich wie Option 1 

Ich würde dann Option 1 wählen
Oder?